app/src/main/java/com/keylesspalace/tusky/fragment/ViewThreadFragment.java

```diff
@@ -191,7 +191,7 @@ public void onResponse(@NonNull Call<Status> call, @NonNull Response<Status> res
                     StatusViewData.Concrete newViewData = viewDataBuilder.createStatusViewData();
 
                     statuses.setPairedItem(position, newViewData);
-                    adapter.setItem(position, newViewData, true);
```

Comment:
I was thinking that this is a bad idea because adapter wouldn't new about the changes and I tried to hack around with ItemAnimator but then I realized that it's actually cool because we still update adapter's collection